main.py

```diff
@@ -79,11 +84,23 @@ def main():
     params.timeout = args.timeout
     params.file = args.file
 
+    ### Debug message toggle ###
+    if args.debug:
+        BoardShim.set_log_level(LogLevels.LEVEL_DEBUG.value)
+
+    ### Board Id selection ###
+    if args.board_id.isdigit():
+        master_board_id = int(args.board_id)
+    else:
+        master_board_id = BoardIds[args.board_id.upper()]
+    
     ### OSC Setup ###
     use_old_reporter = args.use_old_reporter
     ip = args.osc_ip_address
     send_port = args.osc_port
-    osc_reporter = Old_OSC_Reporter(ip, send_port) if use_old_reporter else OSC_Reporter(ip, send_port)
+    reporters = [Old_OSC_Reporter(ip, send_port) if use_old_reporter else OSC_Reporter(ip, send_port)]

```

Comment:
I'm unsure why this needs to be a list, it should be possible to just assign the reporter based on the `use_old_reporter` variable by itself no?

Reply:
its to allow for multiple reporters to work, like the debug reporter, or the future vmc reporter to support vtuber software

Reply:
I see, if that's the case then I'd export this out into its own class which has the sole responsibility of managing the registration and maintenance of individual reporters, the only thing we should ideally be using the main class for going forward should be initialization